src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -2531,6 +2531,7 @@ void ObjectSynchronizer::deflate_common_idle_monitors_using_JT(bool is_global, J
   if (ls != NULL) {
     if (is_global) {
       ls->print_cr("async-deflating global idle monitors, %3.7f secs, %d monitors", timer.seconds(), deflated_count);
+      GVars.stw_random = os::random();
```

Comment:
This restoration of the setting of GVars.stw_random is a left
over from an earlier version of Erik's work on JDK-8247281.
That fix was pushed separately using JDK-8252126.